src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/settings/main.js

```diff
@@ -433,7 +434,7 @@ define([
                 this.sandbox.dom.addClass(constants.externalLink.titleContainer, constants.validateErrorClass);
             }
 
-            if (!data.external) {
+            if (!data.external || !data.urlParts.scheme || !data.urlParts.specificPart) {
```

Comment:
I still think it is strange that we are already duplicating stuff like that in memory already :confused: